### ISSUE TITLE
Summary

### ISSUE THREAD

**Issue by farishkash** [jonboyle/prj-rev-bwfs-dasmoto#4]
Overall met expectations.

Outside of alt text missing, the HTML structure is great.  I didn't see any errors or major issues.

The CSS had some minor issues such as CSS repetition.  The goal is always to type as little as possible when writing CSS.  Font family repetition is one of the most common mistakes here and easy to fix.

To prepare you for the upcoming projects go to flexboxfroggy.com and see how far you can get.  The later projects will use flexbox heavily.
